Src/Campus.Master.API/Startup.cs

```diff
@@ -31,25 +32,35 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            var xmlDocFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
-            var xmlDocPath = Path.Combine(AppContext.BaseDirectory, xmlDocFile);
-            
+            string xmlDocFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
```

Comment:
Why did you change VAR to STRING?

Reply:
My code conventions settings have been applied

Reply:
This settings change var to string in non-obvious cases

Reply:
Wierd conventions, but I'll let you keep 'em for this time)